packages/seal/src/Search/Condition/AbstractGroupCondition.php

```diff
@@ -13,35 +13,21 @@
 
 namespace Schranz\Search\SEAL\Search\Condition;
 
+/**
+ * @internal this class is internal please use AndCondition or OrCondition directly
+ */
 abstract class AbstractGroupCondition
 {
     /**
      * @var array<EqualCondition|GreaterThanCondition|GreaterThanEqualCondition|IdentifierCondition|LessThanCondition|LessThanEqualCondition|NotEqualCondition|AndCondition|OrCondition>
      */
-    private array $conditions = [];
+    public array $conditions = [];
```

Comment:
This will allow to set conditions directly that are not validated against the allowed types. That's why I did not do this.

Reply:
Good catch `readonly` missing :)